### ISSUE TITLE
createAppUser script is not included in the slim image flavor

### ISSUE THREAD

**Issue by vvalencia-cl** [gvenzl/oci-oracle-xe#128]
Hi. 

Using gvenzl/oracle-xe:21.3.0-slim, I was trying to create a second user for my docker-copose container, but when executing createAppUser failed with the message "createAppUser not found".

When changed to gvenzl/oracle-xe:21.3.0, the command wordked without a problem

**Comment by aleybovich** [comment]
I was gonna submit a similar issue - when passing APP_USER and APP_USER_PASSWORD to docker run (when using latest slim version), the user is not getting created. Looks like it's the same issue

**Comment by gvenzl** [comment]
Hey @vvalencia-cl & @aleybovich,

Unfortunately, I cannot reproduce either of these on my end:

@aleybovich, the `APP_USER` & `APP_USER_PASSWORD` case is working for me:

```
[gvenzl@localhost ~]$ podman run --name test -e ORACLE_RANDOM_PASSWORD=y -e APP_USER=test -e APP_USER_PASSWORD=test gvenzl/oracle-xe:21.3.0-slim
CONTAINER: starting up...
CONTAINER: first database startup, initializing...
CONTAINER: uncompressing database data files, please wait...
CONTAINER: done uncompressing database data files, duration: 21 seconds.
CONTAINER: starting up Oracle Database...

LSNRCTL for Linux: Version 21.0.0.0.0 - Production on 20-AUG-2022 16:04:13

Copyright (c) 1991, 2021, Oracle.  All rights reserved.

Starting /opt/oracle/product/21c/dbhomeXE/bin/tnslsnr: please wait...

TNSLSNR for Linux: Version 21.0.0.0.0 - Production
System parameter file is /opt/oracle/homes/OraDBHome21cXE/network/admin/listener.ora
Log messages written to /opt/oracle/diag/tnslsnr/59a730ab3c43/listener/alert/log.xml
Listening on: (DESCRIPTION=(ADDRESS=(PROTOCOL=ipc)(KEY=EXTPROC_FOR_XE)))
Listening on: (DESCRIPTION=(ADDRESS=(PROTOCOL=tcp)(HOST=0.0.0.0)(PORT=1521)))

Connecting to (DESCRIPTION=(ADDRESS=(PROTOCOL=IPC)(KEY=EXTPROC_FOR_XE)))
STATUS of the LISTENER
------------------------
Alias                     LISTENER
Version                   TNSLSNR for Linux: Version 21.0.0.0.0 - Production
Start Date                20-AUG-2022 16:04:13
Uptime                    0 days 0 hr. 0 min. 0 sec
Trace Level               off
Security                  ON: Local OS Authentication
SNMP                      OFF
Default Service           XE
Listener Parameter File   /opt/oracle/homes/OraDBHome21cXE/network/admin/listener.ora
Listener Log File         /opt/oracle/diag/tnslsnr/59a730ab3c43/listener/alert/log.xml
Listening Endpoints Summary...
  (DESCRIPTION=(ADDRESS=(PROTOCOL=ipc)(KEY=EXTPROC_FOR_XE)))
  (DESCRIPTION=(ADDRESS=(PROTOCOL=tcp)(HOST=0.0.0.0)(PORT=1521)))
The listener supports no services
The command completed successfully
ORACLE instance started.

Total System Global Area 1241512272 bytes
Fixed Size		    9685328 bytes
Variable Size		  671088640 bytes
Database Buffers	  553648128 bytes
Redo Buffers		    7090176 bytes
Database mounted.
Database opened.

CONTAINER: Resetting SYS and SYSTEM passwords.

User altered.


User altered.

############################################
ORACLE PASSWORD FOR SYS AND SYSTEM: ODA2NDkz
############################################

Session altered.


User created.


Grant succeeded.


#########################
DATABASE IS READY TO USE!
#########################
```

```
[gvenzl@localhost ~]$ podman exec -ti test bash
bash-4.4$ sqlplus test/test@XEPDB1

SQL*Plus: Release 21.0.0.0.0 - Production on Sat Aug 20 16:07:15 2022
Version 21.3.0.0.0

Copyright (c) 1982, 2021, Oracle.  All rights reserved.


Connected to:
Oracle Database 21c Express Edition Release 21.0.0.0.0 - Production
Version 21.3.0.0.0

SQL>
```

@vvalencia-cl, likewise, the `createAppUser` is working for me, at least via `docker exec`:

```
[gvenzl@localhost ~]$ podman exec test createAppUser test2 test2

Session altered.


User created.


Grant succeeded.

[gvenzl@localhost ~]$ podman exec -ti test sqlplus test2/test2@XEPDB1

SQL*Plus: Release 21.0.0.0.0 - Production on Sat Aug 20 16:13:07 2022
Version 21.3.0.0.0

Copyright (c) 1982, 2021, Oracle.  All rights reserved.

Last Successful login time: Sat Aug 20 2022 16:12:54 +00:00

Connected to:
Oracle Database 21c Express Edition Release 21.0.0.0.0 - Production
Version 21.3.0.0.0

SQL>
```

Can you please share more information on what you are seeing on your end?
